src/Rings/mpoly-graded.jl

```diff
@@ -756,7 +756,7 @@ function singular_poly_ring(R::MPolyDecRing; keep_ordering::Bool = false)
   if !keep_ordering
     return singular_poly_ring(forget_decoration(R), default_ordering(R))
   end
-  return singular_poly_ring(forget_decoration(R), keep_ordering = keep_ordering)
+  return singular_poly_ring(forget_decoration(R); keep_ordering = keep_ordering)
```

Comment:
```suggestion
  return singular_poly_ring(forget_decoration(R); keep_ordering)
```
```suggestion
  return singular_poly_ring(forget_decoration(R); keep_ordering)
```
With the ";" this is no longer needed